unified-codes/apps/web/src/sagas/DetailSaga.ts

```diff
@@ -44,6 +44,18 @@ const getEntityQuery = (code: string) => `
       type
       description
     }
+    parents {
```

Comment:
Intended as a replacement for product fetching. Happy to remove if objections 🤣

Reply:
can an entity have multiple parents? wondering why this is an array

Reply:
As of recent changes, the `product` field  has been made redundant... so might be able to get rid of this entirely anyway? Although, to keep the GraphQL as flexible as possible, just having this field which exposes the reverse edge for `children` is probably a good idea (I don't think there are any entities with multiple parents, but since it's an array in the Dgraph data model, may as well keep it that way for simplicity and future-proofing)?

Reply:
Also just a note that this isn't needed right now, as the web UI only traverses downwards. We definitely want to be able to see parents, and I think updating the `DetailPage` to pre-expand the list to show the entity and it's parents is the easy solution to this. This will require this field, so that's why I don't have an objection to keeping it in there even though we won't be using it right now.

Reply:
Hm yeah, I don't think there is currently a use case for multiple `parents` (although there will potentially be multiple levels of `parent`s above any given node). The schema does allow it though because it's a direct reversal of `children` which is an array 🤔 . I guess we could add parents manually as a separate edge but thought using reverse edge would be simpler 😆